lambda/voting/backend/.env

```diff
@@ -0,0 +1,12 @@
+// Your MetaKeep API key
+API_KEY = 
+
```

Comment:
here also these one-off scripts can be scripts folder.

And in the readme let's describe the folder structure.